### ISSUE TITLE
Error executing 'postInstallation': EROFS: read-only file system

### ISSUE THREAD

**Issue by xcffl** [bitnami/bitnami-docker-wordpress-nginx#12]
**Description**
When using this image in Kubernetes, if you mount the `wordpress-server-block.conf` (whichi is a ConfigMap) to `/opt/bitnami/nginx/conf/server_blocks/wordpress-vhosts.conf`, it will trigger an error which says
> Error executing 'postInstallation': EROFS: read-only file system, chown '/opt/bitnami/nginx/conf/server_blocks/wordpress-vhosts.conf' 

BTW, the example Chart mounts nginx's configuration file into a wrong place.

**Steps to reproduce the issue:**

1. In a Kubernetes cluster
2. Use this chart https://github.com/helm/charts/tree/master/stable/wordpress but claim to use the nginx version image
3. Create a ComfigMap which contains the same content as https://github.com/bitnami/bitnami-docker-wordpress-nginx/blob/master/wordpress-server-block.conf
4. Manually edit the wordpress deployment's yaml, adding
````
      - configMap:
          defaultMode: 436
          name: wordpress-nginx-config
````
and
````
        volumeMounts:
        - mountPath: /opt/bitnami/nginx/conf/server_blocks/wordpress-vhosts.conf
          name: wordpress-nginx-config
          subPath: wordpress-vhosts.conf
````

**Describe the results you received:**

The deployment failed to start, prompting
> Error executing 'postInstallation': EROFS: read-only file system, chown '/opt/bitnami/nginx/conf/server_blocks/wordpress-vhosts.conf' 

**Describe the results you expected:**

The configmap is successfully mounted at that place and the deployment starts.

**Additional information you deem important (e.g. issue happens only occasionally):**

**Version**

- Output of `docker version`:

```
(paste your output here)
```

- Output of `docker info`:

```
(paste your output here)
```

- Output of `docker-compose version` (if applicable):

```
(paste your output here)
```

**Additional environment details (AWS, VirtualBox, Docker for MAC, physical, etc.):**
GKE

**Comment by sameersbn** [comment]
The nginx component attempts to perform some `chown` operations on the `nginx/conf` directory, which fails when a configmap is mounted at the location. We will work on correcting this issue and get back to you  with an update. Thanks for reporting this issue.

**Comment by sameersbn** [comment]
A fix for this is in the works and should be landed soon

**Comment by sameersbn** [comment]
A fix for this issue has been landed and available in `5.3.2-debian-9-r6`.
